CefSharp/Internals/StringCheck.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2020 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What's with the extra license block?

Reply:
The licence was copied from the following file I randomly selected.

https://github.com/cefsharp/CefSharp/blob/940034314b5961f8b80d84d8da7705eaa1dd95e6/CefSharp/Internals/PathCheck.cs#L5

I can submit a separate PR to remove from this page?

Reply:
That file has code copied from the .Net framework source. No need for additional license unless you've used code from somewhere else.